ideas/58-mainnet.md

```diff
@@ -0,0 +1,71 @@
+## Preamble
+
+    Idea: 58-mainnet-default
+    Title: Use Mainnet as the default network
+    Status: In progress
+    Created: 2017-12-08
+
+
+## Summary
+Alpha builds of Status should default to Mainnet. We should be using Status in a real-world environment.
+
+## Swarm Participants
+
+- Lead Contributor: @adambabik
+- Testing & Evaluation: <!-- @username -->
+- Contributor: @themue
+- Contributor: <!-- @username -->
+- PM: @chadyj 
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+Currently the Status alpha builds use testnet, but using Mainnet is a key milestone for the public launch of Status. There will be potentially be major performance and security implications so these issues should be surfaced as soon as possible so the app can be tested in a real-world environment. 
```

Comment:
Always got troubles with weak terms like _"should"_ and _"as soon as possible"_. Better is "have to be surfaced until MVP" with the MVP date mentioned below.

Reply:
I am not sure how I can rephrase it to make it more clear. I understand it this way that this swarm is about enabling mainnet so that all these issues can be surfaced. I am not sure it's gonna happen exactly after MVP or by the end of the swarm.

Reply:
Maybe _"One goal is to surface these issues early enough for testing the app in a real-world environment."_?

Reply:
Thanks!